src/robusta/integrations/receiver.py

```diff
@@ -66,12 +72,18 @@ def start_receiver(self):
         receiver_thread = Thread(target=self.run_forever)
         receiver_thread.start()
 
-    def __run_external_action_request(self, request: ActionRequestBody):
+    def __run_external_action_request(self,
+                                      request: ActionRequestBody,
+                                      sync_response: bool,
+                                      no_sinks: bool,
+                                      ) -> Optional[Dict[str, Any]]:
         logging.info(f"got callback `{request.action_name}` {request.action_params}")
-        self.event_handler.run_external_action(
+        return self.event_handler.run_external_action(
```

Comment:
It's a little hard to track what's actually inside the `return` here. This isn't a must, but at some point it would improve readability if we changed the return type from `Dict[str,Any]` to a class/object.

Reply:
No handling atm